lib/clif-backend/src/relocation.rs

```diff
@@ -8,6 +8,18 @@ use cranelift_codegen::ir::{self, ExternalName, LibCall, SourceLoc, TrapCode};
 use hashbrown::HashMap;
 use wasmer_runtime_core::{structures::TypedIndex, types::LocalFuncIndex};
 
+pub mod call_names {
+    pub const LOCAL_NAMESPACE: u32 = 1;
+    pub const IMPORT_NAMESPACE: u32 = 2;
+
+    pub const STATIC_MEM_GROW: u32 = 0;
+    pub const STATIC_MEM_SIZE: u32 = 1;
+    pub const SHARED_STATIC_MEM_GROW: u32 = 2;
+    pub const SHARED_STATIC_MEM_SIZE: u32 = 3;
+    pub const DYNAMIC_MEM_GROW: u32 = 4;
+    pub const DYNAMIC_MEM_SIZE: u32 = 5;
+}
```

Comment:
Should these be `enums`?

Reply:
These are required to be numbers, so constants are better in this scenario.